### PR TITLE
ZEPPELIN-1374. Should prevent use dot in interpreter name

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
dot is invalid for interpreter name as it is used as the separator of interpreter group name and interpreter name.

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1374

### How should this be tested?
Tested it manually as shown in the screenshot. 

### Screenshots (if appropriate)
image

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
\cc @minahlee @jongyoul  Ready for review.

**Comment by corneadoug** [comment]
We might also want to make sure that the Websocket API and basic API (if one exist), also check the name accordingly

**Comment by zjffdu** [comment]
@corneadoug make sense, I will update the PR

**Comment by jongyoul** [comment]
In backend side, it might be OK.

**Comment by minahlee** [comment]
LGTM

**Comment by minahlee** [comment]
Merge into master and branch-0.6 if there is no further comments

File: zeppelin-web/src/app/interpreter/interpreter.controller.js [hunk 1]
```diff
@@ -325,6 +325,15 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
         return;
       }
 
+      if (!$scope.newInterpreterSetting.name.indexOf('.') >= 0) {
```
Comment (ColZer) [inline zeppelin-web/src/app/interpreter/interpreter.controller.js]: maybe this is a bug!

Comment (ColZer) [inline zeppelin-web/src/app/interpreter/interpreter.controller.js]: hi, maybe this is a bug.
`if ($scope.newInterpreterSetting.name.indexOf('.') >= 0) {`

Comment (zjffdu) [inline zeppelin-web/src/app/interpreter/interpreter.controller.js]: Thanks @ColZer, this is a bug, maybe I introduce this when rebasing.

**Pull request closed without merging**